torch/quantization/ns/numeric_suite_core_apis_fx.py

```diff
@@ -233,7 +232,7 @@ def add_activation_info_to_dict(
             key = mod.ref_name
             if key not in results:
                 results[key] = {}
-            results[key][model_name] = {
+            results[key][mod.model_name] = {
```

Comment:
Dont follow this part, do we expect every module in model.named_modules() to have a model_name attrivute?